### ISSUE TITLE
Jump bug 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dogo/starworms#3]
```
What steps will reproduce the problem?
1. Press CTRL to jump
2. Look the player will sink and don't jump

What is the expected output? What do you see instead?
The player will sink and don't jump

What version of the product are you using? On what operating system?
0.1.7.4-alpha - linux Ubuntu 6.10

Please provide any additional information below.


```

Original issue reported on code.google.com by `diogo.au...@gmail.com` on 30 Jan 2007 at 1:04

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Fixed on 0.1.8.0-alpha.
```

Original comment by `diautilio` on 11 Jan 2008 at 4:33
- Changed state: **Fixed**
